### PR TITLE
Added widget interface.

### PR DESCRIPTION
**Pull request by michal-damiecki**
UIE-315

### CODE REVIEW

File: typings/IWidget.d.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import {JSONSchema4} from 'json-schema';
+import {IUiSchema} from './IUiSchema';
+
+export interface IWidget {
+  schema: {
+    [key: string]: JSONSchema4;
+  };
+  uiSchema: {
```
Comment (stepek) [inline typings/IWidget.d.ts]: IMHO it's should be optional.

File: typings/IWidget.d.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    [key: string]: IUiSchema;
+  };
+  value: any[] | boolean | number | null | object | string;
```
Comment (stepek) [inline typings/IWidget.d.ts]: it's should be optional

File: typings/IWidget.d.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  required: boolean;
```
Comment (stepek) [inline typings/IWidget.d.ts]: its should be optional

**Comment by michal-damiecki** [comment]
@stepek done

**Pull request merged**